pkg/api/mlflow/common/helpers.go

```diff
@@ -1,6 +1,62 @@
 package common
 
+import (
+	"mime"
+	"path"
+	"strings"
+
+	"golang.org/x/exp/slices"
+)
+
+// textTypes used by GetContentType.
+var textTypes = []string{
```

Comment:
please add `.` at the beginning of those extensions so we can refer to them without further manipulation to the result of `path.Ext` — this will be consistent with how the Go stdlib does things

Reply:
will do